File: ToyMachineInterpreterFX/src/View/RunCommand.java
package View;

import Controller.Controller;

public class RunCommand extends Command {
    private Controller ctr;

    public RunCommand(String key, String desc, Controller ctr) {
        super(key, desc);
        this.ctr = ctr;
    }

    @Override
    public void execute() {
        try {
            ctr.setLogging(true);
            ctr.allStep();

        } catch (Exception e) {
            System.out.println(e.getMessage());
        }
    }
}

File: Toy-Language-C#-Intepreter/Toy-Language-CS-Interpreter/Toy-Language-CS-Interpreter/View/RunCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Toy_Language_CS_Interpreter.Controller;
namespace Toy_Language_CS_Interpreter.View
{
    class RunCommand:Command
    {
        private Controller.Controller controller;

        public RunCommand(string key, string description, Controller.Controller controller) : base(key,description)
        {
            this.controller = controller;
        }

        public override void Execute()
        {
            try
            {
                controller.ExecuteAllSteps();
            }
            catch (Exception e)
            {
                Console.WriteLine(e.Message);
            }
        }
    }
}
